### PR TITLE
MAP-407-Add integration tests reception move journey

### PR DESCRIPTION
**Pull request by davidthomas22**

### CODE REVIEW

File: integration-tests/integration/receptionMove/receptionConfirmMovePage.cy.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      {
+        alertId: 6,
+        alertType: 'X',
+        alertTypeDescription: 'Security',
+        alertCode: 'XGANG',
+        alertCodeDescription: 'Gang Member',
+        dateCreated: '2023-10-10',
+        expired: false,
+        active: true,
+        addedByFirstName: 'GURNANK',
```
Comment (GurnankCheema) [inline integration-tests/integration/receptionMove/receptionConfirmMovePage.cy.js:28]: I'f prefer for my name not to be in tests!

Comment (davidthomas22) [reply integration-tests/integration/receptionMove/receptionConfirmMovePage.cy.js:28]: Good spot, thanks

File: integration-tests/integration/receptionMove/receptionConsiderRisksPage.cy.js [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+          lastName: 'BLANCYANA',
+          agencyId: 'MDI',
+        },
+      },
+    ],
+  })
+  cy.task('stubOffendersInReception', {
+    agencyId: 'MDI',
+    inReception: [
+      {
```
Comment (GurnankCheema) [inline integration-tests/integration/receptionMove/receptionConsiderRisksPage.cy.js:245]: I don't think we use a lot of the data in each prisoners object here. Can it be limited to just the bits used in the code (i.e name, offenderNo, alerts). This is just so the test files aren't unnecessarily large

File: integration-tests/integration/receptionMove/receptionConsiderRisksPage.cy.js [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+    page
+      .form()
+      .moveReason()
+      .type('{backspace}{backspace}{backspace}{backspace}Transfer due to staff conflict of interest')
+    page.form().submitButton().click()
+    cy.title().should('eq', 'This person has been moved to reception - Digital Prison Services')
+  })
+})
+
+context.skip('Reception full journey', () => {
```
Comment (GurnankCheema) [inline integration-tests/integration/receptionMove/receptionConsiderRisksPage.cy.js:341]: test is being skipped

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**